packages/airnode-examples/integrations/coingecko-signed-data/README.md

```diff
@@ -0,0 +1,28 @@
+# Coingecko-signed-data example integration
+
+This example contains the same API integration as `../coingecko`.
+
+The only difference is that this integration allows the endpoint to return signed data that can be submitted on chain by
+the to update a beacon.
```

Comment:
This sentence seems to be missing a word :thinking:

Reply:
Yes, looks like it. Shorten it to just
`The only difference is that this integration allows the endpoint to return signed data that can be then submitted on chain.`